src/foraging_gui/Foraging.py

```diff
@@ -923,7 +958,14 @@ def _GetSettings(self):
             'newscale_serial_num_box4':'',
             'show_log_info_in_console':False,
             'default_ui':'ForagingGUI.ui',
-            'open_ephys_machine_ip_address':''
+            'open_ephys_machine_ip_address':'',
+            'metadata_dialog_folder':os.path.join(self.SettingFolder,"metadata_dialog")+'\\',
+            'rig_metadata_folder':os.path.join(self.SettingFolder,"rig_metadata")+'\\',
+            'project_infor_file':os.path.join(self.SettingFolder,"Project Name and Funding Source v2.csv"),
```

Comment:
What is the `project_infor_file`? and is that a typo ("info")?

Reply:
It defines the project excel file required to automatically load the project information to generate project description. It should be stored in the ForagingSettings folder by default (Or specified in the ForagingSettings.json). 

I will update the name to `project_info_file`.